ts/Stock/StockToolsGui.ts

```diff
@@ -284,6 +285,7 @@ setOptions({
                 verticalLabel: 'Vertical label',
                 verticalArrow: 'Vertical arrow',
                 fibonacci: 'Fibonacci',
+                fibonacciTimeZones: 'Fibonacci Time Zones',
```

Comment:
Is it `Time Zone` or `Time Zones` (plural)? I can see different variants.

@pawelfus  Could you please help with the decision?
IMO it should be `Fibonacci Time Zones`

After the decision, please unify it everywhere.

Reply:
Plural, I'll look for them and correct, thanks

Reply:
Remember about changing the name of the `fibonacci-timezone.svg` file ;)

Reply:
Seems like only the `svg` icon is not in the plural form